gcs/CHANGES.md

```diff
@@ -82,6 +82,8 @@
     fs.gs.grpc.read.zerocopy.enable (default : true)
     ```
 
+1.  Added dependency on new storage client library: [google-cloud-storage](https://github.com/googleapis/java-storage/tree/main/google-cloud-storage).
```

Comment:
why are we adding this here? This should be added after the "next" tag on the top